src/fiatconnect/FiatDetailsScreen.test.tsx

```diff
@@ -103,157 +143,230 @@ describe('FiatDetailsScreen', () => {
-      fiatAccountSchema: quoteWithAllowedValues.getFiatAccountSchema(),
-    })
-  })
-  it('button remains disabled if required input field is empty', () => {
-    const { getByText, getByTestId, queryByTestId } = render(
-      <Provider store={store}>
-        <FiatDetailsScreen {...mockScreenPropsWithAllowedValues} />
-      </Provider>
-    )
-
-    expect(getByText('fiatAccountSchema.institutionName.label')).toBeTruthy()
-    expect(getByText('fiatAccountSchema.accountNumber.label')).toBeTruthy()
-    expect(queryByTestId(/errorMessage-.+/)).toBeFalsy()
-
-    fireEvent.changeText(getByTestId('input-accountNumber'), fakeAccountNumber)
-
-    expect(getByTestId('submitButton')).toBeDisabled()
-  })
-  it('shows validation error if the input field does not fulfill the requirement after delay', () => {
-    const { getByText, getByTestId, queryByTestId } = render(
-      <Provider store={store}>
-        <FiatDetailsScreen {...mockScreenPropsWithAllowedValues} />
-      </Provider>
-    )
-
-    expect(getByText('fiatAccountSchema.institutionName.label')).toBeTruthy()
-    expect(getByText('fiatAccountSchema.accountNumber.label')).toBeTruthy()
-    expect(queryByTestId(/errorMessage-.+/)).toBeFalsy()
-
-    fireEvent.changeText(getByTestId('input-accountNumber'), '12dtfa')
-
-    // Should see an error message saying the account number field is invalid
-    // after delay
-    expect(queryByTestId(/errorMessage-.+/)).toBeFalsy()
-    jest.advanceTimersByTime(1500)
-    expect(getByTestId('errorMessage-accountNumber')).toBeTruthy()
-    expect(getByText('fiatAccountSchema.accountNumber.errorMessageDigit')).toBeTruthy()
-    expect(getByTestId('submitButton')).toBeDisabled()
-  })
-  it('shows validation error if the input field does not fulfill the requirement immediately on blur', () => {
-    const { getByText, getByTestId, queryByTestId } = render(
-      <Provider store={store}>
-        <FiatDetailsScreen {...mockScreenProps} />
-      </Provider>
-    )
+    ) => {
+      it('can view a list of bank fields', () => {
+        const { getByText, getByTestId, queryByTestId } = render(
+          <Provider store={store}>
+            <FiatDetailsScreen {...screenProps} />
+          </Provider>
+        )
+
+        expect(getByText('fiatAccountSchema.institutionName.label')).toBeTruthy()
+        // checks presence of picker, testID is hardcoded and not customizable
+        expect(getByTestId('android_touchable_wrapper')).toBeTruthy()
+
+        expect(getByText(`fiatAccountSchema.${accountSchema}.label`)).toBeTruthy()
+        expect(getByTestId(`input-${input}`)).toBeTruthy()
+
+        expect(queryByTestId(/errorMessage-.+/)).toBeFalsy()
+
+        expect(getByTestId('submitButton')).toBeTruthy()
+        expect(getByTestId('submitButton')).toBeDisabled()
+        expect(getByText('fiatDetailsScreen.submitAndContinue')).toBeTruthy()
+      })
+      it('renders header with provider image', () => {
```

Comment:
I think header / cancel button tests independent of the schema doesn't need to be tested for both schemas